packages/fd.rb

```diff
@@ -1,42 +1,33 @@
 require 'package'
 
 class Fd < Package
-  description "A simple, fast and user-friendly alternative to 'find'"
-  homepage 'https://github.com/sharkdp/fd'
-  version '8.3.0'
-  license 'Apache License Version 2.0'
+  description "A simple, fast and user-friendly alternative to 'find'."
+  homepage 'https://github.com/sharkdp/fd/'
+  @_ver = '8.3.0'
+  version @_ver + '-1'
+  license 'Apache-2.0 and MIT'
   compatibility 'all'
-  source_url ({
-    aarch64: 'https://github.com/sharkdp/fd/releases/download/v8.3.0/fd-v8.3.0-arm-unknown-linux-musleabihf.tar.gz',
-     armv7l: 'https://github.com/sharkdp/fd/releases/download/v8.3.0/fd-v8.3.0-arm-unknown-linux-musleabihf.tar.gz',
-       i686: 'https://github.com/sharkdp/fd/releases/download/v8.3.0/fd-v8.3.0-i686-unknown-linux-musl.tar.gz',
-     x86_64: 'https://github.com/sharkdp/fd/releases/download/v8.3.0/fd-v8.3.0-x86_64-unknown-linux-musl.tar.gz',
-  })
-  source_sha256 ({
-    aarch64: '14646b148e3cfbdaae667b32cf5eea48cf11e6ddc3dc0d845177c13053a316ff',
-     armv7l: '14646b148e3cfbdaae667b32cf5eea48cf11e6ddc3dc0d845177c13053a316ff',
-       i686: '279038bf1a9703934e3bddf2fa359af1445d3bbe4ab80e758ab5c58d96b0dd0e',
-     x86_64: 'ff36894153f352a29ecf0efd97adcd3ac788053801726861c6db4bdec1aa8461',
-  })
+  source_url 'https://github.com/sharkdp/fd.git'
+  git_hashtag 'v' + @_ver
+
+  depends_on 'rust' => :build
 
   def self.build
-    File.write('fd.bash', "source #{CREW_PREFIX}/share/fd/autocomplete/fd.bash-completion")
+    system 'cargo build --release'
   end
 
   def self.install
-    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"
-    FileUtils.mkdir_p "#{CREW_DEST_MAN_PREFIX}/man1"
-    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share/fd"
-    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/bash.d"
-    FileUtils.mv 'autocomplete', "#{CREW_DEST_PREFIX}/share/fd"
-    FileUtils.install 'fd', "#{CREW_DEST_PREFIX}/bin/fd", mode: 0o755
-    FileUtils.install 'fd.1', "#{CREW_DEST_MAN_PREFIX}/man1/fd.1", mode: 0o644
-    FileUtils.install 'fd.bash', "#{CREW_DEST_PREFIX}/etc/bash.d/fd.bash", mode: 0o644
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin/"
+    FileUtils.mkdir_p "#{CREW_DEST_MAN_PREFIX}/man1/"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share/bash-completion/completions/"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share/zsh/site-functions/"
+    FileUtils.install "target/release/fd", "#{CREW_DEST_PREFIX}/bin/fd", mode: 0o755
+    FileUtils.install "target/release/build/fd-find-88bf4d216dc4818c/out/fd.bash", "#{CREW_DEST_PREFIX}/share/bash-completion/completions/fd"
+    FileUtils.install "doc/fd.1", "#{CREW_DEST_MAN_PREFIX}/man1/fd.1"
+    FileUtils.install "contrib/completion/_fd", "#{CREW_DEST_PREFIX}/share/zsh/site-functions/_fd"
   end
 
-  def self.postinstall
-    puts "\nTo enable bash completion, execute the following:".lightblue
-    puts "source ~/.bashrc".lightblue
-    puts "\nType 'fd -h' to get started.\n".lightblue
```

Comment:
You could keep this last line to help users get started.

Reply:
it feels unnecessary. the binary is the same name as the package, and --help is the standard help command.

Reply:
I don't think it hurts and it probably is a bit pretentious to assume the user knows how to execute.

Reply:
I disagree. we provide manpages and the `crew files` commands for this kind of lookup information. plus, fd isn't geared toward desktop (likely less experienced) users anyway. @satmandu third opinion?

Reply:
I'm with @saltedcoffii on this one. I think that telling people to use `fd -h` is ob

Reply:
I'm with @saltedcoffii on this one. We have already added a man page, so we don't need to suggest people use `fd -h`.

Reply:
Awesome, let's get some binaries built 🎉